e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,101 @@
+print_prefix="E2E: BATCH MODE TEST:"
```

Comment:
```suggestion
#!/usr/bin/env bash

print_prefix="E2E: BATCH MODE TEST:"
```

Reply:
I actually think this might be why Travis is failing. I see logging messages like this:

```
e2e-tests/batch-mode-tests.sh: 6: e2e-tests/batch-mode-tests.sh: [[: not found
```

On some versions of Linux (and often in usermode) `sh` is symlinked to `/bin/bash`, but there are a few cases where things will fall back to old-school `sh` (or at least bash running in old-school `sh` mode) where `[[` conditional aren't supported.

Reply:
finally this sank in, thanks @ibacher